client/src/pages/CreateSurvey/CreateSurvey.test.jsx

```diff
@@ -17,8 +17,10 @@ describe('Form works as expected', () => {
 
     const titleInput = inputs[0];
 
-    fireEvent.change(titleInput, { target: { value: 'a' } });
+    const longString = 'a'.repeat(61);
```

Comment:
IMO defining the variable is unnecessary as it's definition is concise enough to drop straight into the fireEvent and it's perfectly readable. Happy to be convinced otherwise